src/ModelInterface.py

```diff
@@ -14,12 +13,13 @@
 
 
 class ModelInterface(pl.LightningModule):
-    def __init__(self, model_name):
+    def __init__(self, model_name, optimizer_func, lr):
```

Comment:
I finally used a variable. But I didn't find some ways to enable learning rate schedulers.